build/repo.targets

```diff
@@ -35,9 +36,12 @@
     </GetOSPlatform>
 
     <PropertyGroup>
-      <OutputZip Condition="'$(OSPlatform)'=='Windows'">$(ArtifactsDir)Build.RuntimeStore.win-$(VersionSuffix).zip</OutputZip>
-      <OutputZip Condition="'$(OSPlatform)'=='Linux'">$(ArtifactsDir)Build.RuntimeStore.linux-$(VersionSuffix).zip</OutputZip>
-      <OutputZip Condition="'$(OSPlatform)'=='macOS'">$(ArtifactsDir)Build.RuntimeStore.osx-$(VersionSuffix).zip</OutputZip>
+      <OutputZipSufix Condition="'$(OSPlatform)' == 'Windows'">win</OutputZipSufix>
+      <OutputZipSufix Condition="'$(OSPlatform)' == 'Linux'">linuxp</OutputZipSufix>
```

Comment:
linux not linuxp, also why are we removing the version suffix? It's usually pretty useful to know which packages the store corresponds to.